0001-short-term-release-process/0001-release-process-short-term.md

```diff
@@ -0,0 +1,325 @@
+We would use the collective version number for a single Github Release. The assets for the release
+would be the combined binary packages for each platform. The changelog can also be nicely applied to
+this combined release.
+
+#### Uncertainties
+
+* Should `faucet` and `sn_auditor` be part of a package targeted at users?
+* Should Semantic Versions be dropped from user-facing elements?
+* What are the `X` and `Y` in the version number?
+
+### Alpha Releases
+
+An alpha release would accommodate a scenario in which we wanted to put out quick, experimental code
+to have community users test something on an isolated network. We can branch it off `main` and
+discard the branch when it's done. The branch is intended to have a very short duration. It will be
+possible to apply fixes to it and do a new release on the same branch, in which case the pre-version
+specifier will be incremented. If the fixes are good, we can cherry pick the fix commits back into
+`main`.
+
+An owner should be designated to the whole experiment. They will produce the release, deploy it, and
+coordinate with users.
+
+#### Process
+
+* Prepare a light description that communicates the purpose of the release; we do not need a full
+  changelog for this type of release.
+* Create and checkout an `YYYY-MM-DD-alpha.1` branch from `main`.
+* Use `release-plz update` to bump crate versions.
+* Use a script to apply the `alpha` pre-release specifier to each bumped crate.
+* Create a `chore(release): alpha-YYYY-MM-DD` commit.
+* Push that branch to the upstream `maidsafe` repo, which will kick off the release workflow.
+* The release workflow will produce a public `pre-release` on Github, but the crates will *not* be
+  published.
+* Manually edit the Github Release to provide the description prepared in the first step.
+* Use the `Launch Network` workflow to deploy the `alpha` binaries to an isolated network. *
+* Announce the availability of the binaries to the community. Users can use `safenode-manager`
+  and/or `safeup` with `--version` arguments to obtain the alpha binaries.
+* Perform testing
+* If a problem is identified, there is an opportunity for a small fix/test cycle:
+    - On the same alpha branch, apply the fix.
+    - Use `cargo release version alpha --package X --package Y` to increment the necessary crates to
+      `alpha.2`
+    - Deploy the fix either using an upgrade or by launching a new testnet
+    - Users can test
+    - Repeat if necessary (in practice we should not have many of these)
+* If need be, any fix commits applied on the alpha branch should be cherry picked into `main`.
+* The experiment is over and the branch should be deleted.
+
+The branch is discarded because we don't want the alpha version bumps back in `main`. Crates were
+also not published. The Github Release will function as the historical record of the existence of
+the alpha release.
+
+#### Uncertainties
+
+* Would we need to use the `NETWORK_VERSION` compile-time variable for this?
```

Comment:
I think we should perhaps scrap this.

internal testnets won't cross over (unless we are not careful...but that would be mitigated by:)

We should work our way for the environment released into to have differing genesis keys, and that will in effect prevent this.

Reply:
OK, if we're confident there will be no interference, I will remove this.